crates/sui-indexer-alt-graphql/src/api/types/gas_effects.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::Object;
+use sui_types::{
+    effects::{TransactionEffects as NativeTransactionEffects, TransactionEffectsAPI},
+    gas::GasCostSummary as NativeGasCostSummary,
+};
+
+use crate::{
+    api::{scalars::big_int::BigInt, types::address::Address},
+    error::RpcError,
+    scope::Scope,
+};
+
+use super::object::Object;
+
+pub(crate) struct GasCostSummary {
```

Comment:
Note that we already have a `GasCostSummary` type here:

https://github.com/MystenLabs/sui/blob/8f98f1f7dd992de57e47e48dc6bd8241e4f3501f/crates/sui-indexer-alt-graphql/src/api/types/gas.rs#L9-L11

Probably you want to merge this module with that module.